.github/workflows/benchmark-flow.yml

```diff
@@ -0,0 +1,117 @@
+name: Run a Benchmark Flow
+
+on:
+  workflow_call:
+    inputs:
+      container:
+        type: string
+        default: redisfab/rmbuilder:6.2.7-x64-bullseye
+      module_path:
+        type: string
+        default: bin/linux-x64-release/src/redisgraph.so
+      profile_env:
+        type: number # for default of 0
+      cluster_env:
+        type: string # for default of ""
+      benchmark_glob:
```

Comment:
not sure what`benchmark_glob` stands for

Reply:
Added the comment explaining it. If in the future we need to split tests further in a more precise manner we can use this option.